sencha-workspace/SlateDemonstrationsTeacher/app/controller/Dashboard.js

```diff
@@ -12,17 +12,26 @@ Ext.define('SlateDemonstrationsTeacher.view.DashboardController', {
 
         'Slate.cbl.view.teacher.skill.OverviewWindow',
         'Slate.cbl.view.teacher.skill.OverrideWindow',
-        'Slate.cbl.view.teacher.demonstration.EditWindow'
+        'SlateDemonstrationsTeacher.view.Dashboard',
+        'Slate.cbl.view.teacher.demonstration.EditWindow',
```

Comment:
@aliyeysides global controllers should use their special `views` config to load views which applies smart name resolution and creates a getter for the constructor for you.

````
views: [
    'Dashboard'
]
```
will load `SparkClassroomteacher.view.Dashboard` and then create a getter `this.getDashboardView()` so you can use `this.getDashboardView.create({...})` instead of `Ext.create('SparkClassroomteacher.view.Dashboard', {...})`

For views being loaded from packages instead of within the app you can use `@` to specify the full namespace:

```
views: [
    'skill.OverviewWindow@Slate.cbl.view.teacher'
]
```
will generate the getter `this.getSkillOverviewWindowView()`, it's up to you which parts of the name are before/after the @ based on what makes a nice getter